iis-wf/iis-wf-transformers/src/main/resources/eu/dnetlib/iis/wf/transformers/idreplacer/oozie_app/lib/scripts/replacer_1_field.pig

```diff
@@ -3,7 +3,7 @@ define avro_load_input AvroStorage('$schema_input_main');
 define avro_load_id_mapping AvroStorage('$schema_input_id_mapping');
 
 
-define avro_store_output AvroStorage('$schema_output');
+define avro_store_output AvroStorage('$schema_output', '-doublecolons');
```

Comment:
OK, so the problem was double colon which have to be translated to double underscores?